packages/sd-jwt/tests/sdJwtVc.e2e.test.ts

```diff
@@ -0,0 +1,145 @@
+import type { Key } from '@aries-framework/core'
+
+import { AskarModule } from '@aries-framework/askar'
+import {
+  Agent,
+  DidKey,
+  DidsModule,
+  KeyDidRegistrar,
+  KeyDidResolver,
+  KeyType,
+  TypedArrayEncoder,
+  utils,
+} from '@aries-framework/core'
+import { ariesAskar } from '@hyperledger/aries-askar-nodejs'
+
+import { agentDependencies } from '../../core/tests'
+import { SdJwtModule } from '../src/SdJwtModule'
+
+const getAgent = (label: string) =>
+  new Agent({
+    config: { label, walletConfig: { id: utils.uuid(), key: utils.uuid() } },
+    modules: {
+      sdJwt: new SdJwtModule(),
+      // @ts-ignore
```

Comment:
What's the problem here that required a ts-ignore? It's about not using askar-nodejs as direct dependency?

Reply:
The expected ariesAskar type did not match the one from the module as I think it used 0.2.0-dev.1 and I used 0.1.0. I think I can remove it again